File: app/src/main/java/com/example/todaysshow/object/HomeShow.java
package com.example.todaysshow.object;

public class HomeShow {
    private int imageResouce;
    private String showName;

    public HomeShow(int imageResouce, String showName){
        this.imageResouce = imageResouce;
        this.showName = showName;
    }

    public int getImageResouce() {
        return this.imageResouce;
    }
    public String getShowName(){ return this.showName;}
}

File: app/src/main/java/com/example/todaysshow/object/HomeShow.kt
package com.example.todaysshow.`object`

class HomeShow(imageSource: Int, showName: String) {
    private var imageSource : Int = imageSource
    private var showName : String = showName

    fun getImageResource() : Int{
        return this.imageSource
    }

    fun getShowName(): String{
        return this.showName
    }

}